src/main/docker/startup/startup.sh

```diff
@@ -20,6 +20,9 @@ log() {
     echo "[$(date +%H:%M:%S.%3NZ) #$(printf '%03X\n' $$).??? INFO  -            -   ] ${0##*/}: $@"
 }
 
+# Export file based secrets
+exec $(dirname "$0")/../scripts/export-file-based-secrets.sh
```

Comment:
Does this actually work?  I would have thought that using `exec` would cause the rest of the script to not be executed?